Shared/Index.lua

```diff
@@ -1,9 +1,3 @@
-local sCmd = "luarun"
-
-local function isLuaRun(sInput)
-    return sInput:sub(0, (#sCmd + 1)) == (sCmd.." ")
-end
-
-Console.RegisterCommand( sCmd, function( ... ) -- Each word is an argument
+Console.RegisterCommand( "luarun", function( ... ) -- Each word is an argument
```

Comment:
Could you please keep the previous syntax by removing extra spaces and just keeping the ones between args?

Reply:
I'll do it and then I'll kill you